.github/workflows/sync.yml

```diff
@@ -0,0 +1,195 @@
+name: Sync
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+env:
+  BOT_USERNAME: 'exercism-bot'
+  GIT_USERNAME: 'Exercism Bot'
+  GIT_EMAIL: '66069679+exercism-bot@users.noreply.github.com'
+
+jobs:
+  fetch-tracks:
+    name: Fetch & fork tracks
+    runs-on: ubuntu-20.04
+
+    outputs:
+      tracks: ${{ steps.fetch-tracks.outputs.result }}
+
+    steps:
+    - name: Fetch list of tracks
+      id: fetch-tracks
+      uses: actions/github-script@47f7cf65b5ced0830a325f705cad64f2f58dddf7
+      with:
+        script: |
+          const trackRepos = await github.paginate(github.search.repos, {
+            q: 'org:exercism+topic:exercism-track'
+          })
+
+          // research_experiment_1 is not a real track, don't treat it as one
+          //return trackRepos.flatMap(({ full_name }) => full_name === 'exercism/research_experiment_1' ? [] : [full_name])
```

Comment:
I've just removed the label from this track (cc @ErikSchierboom for your scripts too)

Reply:
Ah nice